src/build/strings.ts

```diff
@@ -156,6 +156,14 @@ const strings = {
   'node.stopOnEntry.description': 'Automatically stop program after launch.',
   'node.timeout.description':
     'Retry for this number of milliseconds to connect to Node.js. Default is 10000 ms.',
+
+  'configuration.warnOnLongPrediction':
+    'Whether a loading prompt should be shown if breakpoint prediction takes a while.',
+  'longPredictionWarning.message':
+    "It's taking a while to configure your breakpoints. You can speed this up by updating the 'outFiles' in your launch.json.",
```

Comment:
Is `outFiles` used right now?

Reply:
It is with this change; we now use that as the pattern grep for, rather than the strategy where we looked for all .js files except for node_modules wherever we had a breakpoint set.

Reply:
Ok I missed that

Reply:
We might also want to say that if outFiles is set, we don't set the instrumentation breakpoint, could help with slowness in that scenario

Reply:
Yea. That could help speed up the sliver of intersection between browsers serving filesystem paths (where vscode lies)  pretty dramatically.